akka-docs/src/main/paradox/stream/operators/Source-or-Flow/flatMapPrefix.md

```diff
@@ -0,0 +1,37 @@
+# flatMapPrefix
```

Comment:
I proposed a few changes to the scaladoc of `flatMapPrefix`, these should probably be propagated here as well where applicable.